db/migrate/20190721113740_create_restaurants.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRestaurants < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
Also, I HIGHLY advise on watching this video: https://www.youtube.com/watch?v=XjPY2D88JIg&feature=youtu.be

It's Tom Clark - a Principal Engineer who talks there about safe migrations.

One thing I would advise on doing is to use `up` and `down` instead of `change`
You don't need to watch the whole video **now** but I would advise on watching it since its really interesting